### DISCUSSION TITLE
PSADT in Constrained Language Mode

### DISCUSSION THREAD

**Discussion by Fixfinder** [PSAppDeployToolkit/PSAppDeployToolkit#1041] [Q&A] [answered]
in powershell Constrained Language mode, it restricts the PSADT script to run and no longer works.
is there a way that enable Full Language mode in PSADT, effectively bypassing Constrained Language?
If not, is there a plan to have a solution in the next release version?

**Comment by mjr4077au** [comment, marked as answer]
It's not up to us to do anything potentially malicious to bypass your security configuration. You should be ensuring your AppLocker configuration permits PSADT from running where you're calling it from.
